libraries/NetworkUtils.py

```diff
@@ -5,13 +5,17 @@
 from libraries.provision.ansible_runner import AnsibleRunner
 
 from keywords.exceptions import ProvisioningError
+from keywords.utils import log_info
 
 
 class NetworkUtils:
 
     def list_connections(self):
-        output = subprocess.check_output("netstat -ant | awk '{print $6}' | sort | uniq -c | sort -n", shell=True)
-        print(output)
+        log_info("\nSocket usage on mobile-testkit client ...")
```

Comment:
Makes it harder to read and more cryptic, just wondering if there is an easy way to avoid this